tests/filecheck/backend/riscv/prologue_epilogue_insertion.mlir

```diff
@@ -30,9 +30,9 @@ riscv_func.func @main() {
   // CHECK-NEXT: fld %{{.*}}, 0
   // CHECK-SAME: (!riscv.reg<sp>) -> !riscv.freg<fs2>
   // CHECK-NEXT: lw %{{.*}}, 8
-  // CHECK-SAME: (!riscv.reg<sp>) -> !riscv.freg<s5>
+  // CHECK-SAME: (!riscv.reg<sp>) -> !riscv.reg<s5>
```

Comment:
This one I can't wrap my head around; how did it not fail before ? Are CHECK-SAME handled at all by our filecheck?

Reply:
Can confirm `CHECK-SAME` is not handled by old filecheck. Would only merge this PR when we are immediately ready to merge the switch to new filecheck.